testing/jormungandr-integration-tests/src/common/jcli/services/transaction_builder.rs

```diff
@@ -73,11 +76,13 @@ impl TransactionBuilder {
         sender: &Wallet,
         input_amount: Value,
         receiver: &Wallet,
+        expiry_date: BlockDate,
```

Comment:
Why even have this function that triggers a lint, when all it does is to invoke the proper builder API?

Reply:
Well, that is a valid question. Especially given that it is used only 2 times.